src/hooks/useMyTickets.ts

```diff
@@ -0,0 +1,24 @@
+import { UserContext } from '@context/usercontext';
+import useAllTickets from '@hooks/useAllTickets';
+import { useContext } from 'react';
+import useSWR from 'swr';
+
+export default function useMyTickets() {
+  const {
+    data: tickets, isLoading: loadingAllTickets, isValidating,
+  } = useAllTickets();
+  const { user } = useContext(UserContext);
+
+  const { data, isLoading: loadingMyTickets } = useSWR(
+    () => (!loadingAllTickets && !isValidating) && 'dataService/myTickets',
```

Comment:
The various use*Tickets SWR hooks take a dependency on `isValidating` from useAllTickets so that they will also refresh whenever useAllTickets (key `dataservice/alltickets`) is mutated